template/deployments/helm/charts/values.yaml

```diff
@@ -60,4 +60,7 @@ tolerations: []
 
 affinity: {}
 
-generatorLabel: {{.GENERATORLABEL}}
\ No newline at end of file
+generatorLabel: {{.GENERATORLABEL}}
+
+# environment variables will be read from the configmap.yaml
+configMapName: configmap
```

Comment:
we dont want the configmap name in here, we want the environment variables defined in here